ArduPlane/quadplane.cpp

```diff
@@ -2634,6 +2634,8 @@ void QuadPlane::vtol_position_controller(void)
 
         // use input shaping and abide by accel and jerk limits
         pos_control->input_vel_accel_NE_m(target_speed_ne_ms, target_accel_ne_mss);
+        // During POS1, we only want to control velocity and acceleration
+        pos_control->stop_pos_NE_stabilisation();
```

Comment:
@lthall Do you see any problem with us using this combination of methods? It does seem a little odd that were calling `input_vel_accel_NE_m` which does integrate position and then zeroing the error in `stop_pos_NE_stabilisation` I wonder if we would be better to add a method that does not integrate position in the fist place.

Reply:
No, I made this function available for this specific purpose. Here is where it is used in the same way in Copter Guided:

https://github.com/ArduPilot/ardupilot/blob/6581d3d0a8567c4d9d43a2d7dfe8023858fba1b5/ArduCopter/mode_guided.cpp#L840